app/_how-tos/discover-and-govern-apis-with-service-catalog.md

```diff
@@ -0,0 +1,138 @@
+        * Integration Admin
+        * Scorecard Admin
+        * Service Admin
+      icon_url: /assets/icons/kogo-white.svg
+    - title: AWS API Gateway
+      include_content: prereqs/service-catalog-iam
+      icon_url: /assets/icons/aws.svg
+    - title: SwaggerHub
+      content: |
+        You must have a [SwaggerHub API key](https://app.swaggerhub.com/settings/apiKey) to authenticate your SwaggerHub account with {{site.konnect_short_name}}. 
+        Additionally, you'll need an [API version](https://support.smartbear.com/swaggerhub/docs/en/manage-apis/versioning.html?sbsearch=API%20Versions0) in SwaggerHub to pull into {{site.konnect_short_name}} as a resource. You can name your SwaggerHub API version whatever you'd like. In this tutorial, we'll refer to your SwaggerHub API version as `swaggerhub-api`.
+      icon_url: /assets/icons/third-party/swaggerhub.svg
+    - title: GitHub
+      content: |
+        To integrate GitHub with Service Catalog, you need the following:
+        * Sufficient permissions in GitHub to authorize third-party applications and install the {{site.konnect_short_name}} GitHub App
+        * A GitHub organization
+        * A repository that you want to pull in to {{site.konnect_short_name}}. You can grant access to either all repositories or selected repositories during the authorization process. You can name your GitHub repository whatever you'd like. In this tutorial, we'll refer to your GitHub repository as `github-repo`.
+        
+        The {{site.konnect_short_name}} app can be managed in your GitHub account under **Applications > GitHub Apps**.
+      icon_url: /assets/icons/github.svg
+tools:
+  - deck
+---
+
+## Configure the AWS API Gateway, GitHub, and SwaggerHub integrations
+
+In this tutorial, we'll be discovering APIs and API specs from AWS API Gateway, GitHub, and SwaggerHub in Service Catalog. Service Catalog allows you to find and centrally view all APIs from these integrations and map them to a Service Catalog service. 
+
+Before you can discover APIs in Service Catalog, you must configure the third-party integrations.
+
+### Configure the AWS API Gateway integration
+
+1. In the {{site.konnect_short_name}} sidebar, click **Service Catalog**.
+1. In the Service Catalog sidebar, click **[Integrations](https://cloud.konghq.com/us/service-catalog/integrations)**. 
+1. Click **AWS API Gateway**.
+1. Click **Add AWS API Gateway instance**.
+1. From the **AWS region** dropdown, select "US East (Ohio)".
+   If you're using a different region, select your region from the dropdown.
+1. In the **IAM role ARN** field, enter the [IAM role you configured for Service Catalog](#aws-api-gateway).
+1. In the **Display name** field, enter `aws-api-gateway-test`.
+1. In the **Instance name** field, enter `aws-api-gateway-test`.
+1. Click **Save**.
```

Comment:
I think this could be an include if you're using the same steps in all three how-tos